downstream/modules/platform/ref-guidelines-hosts-groups.adoc

```diff
@@ -20,7 +20,7 @@ If you use one value in `[database]` and both {Controllername} and {HubNameMain}
 ====
 
 .{ControllerNameStart}
-* {ControllerNameStart} does not configure replication or failover for the database that it uses. 
+* {ControllerNameStart} will not configure replication or failover for the database that it uses, although automation controller should work with any replication that you have. 
 {ControllerName} works with any replication that you have.
```

Comment:
Did you mean to remove the final sentence here since you pretty much said the same thing at the end of the previous sentence above? The only other thing I can say is that typically we try to avoid using future tense and use present tense words like "does" because it presents the message in a more direct way. I'll leave it up to you to whether you want to keep it with "does" instead of "will". 

For reference about future tense, here's a link: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense. It basically says write in present tense as much as possible, except where you cannot avoid it.